UKHomeOffice/nmsw-ui
# NMSW-1124 Create checkbox form input

# Ticket

NMSW-1124

----

## AC

Create a checkbox form component (will submit on button click for now until NMSW-1057)

----

## To test

- run app
- sign in
- click on a declaration
- upload passengers
- on upt page
-> you should see a checkbox component
-> You should be able to select and unselect options
- click submit
-> You should see the form data logged in the console


----

## Notes
